apollo-runtime/src/main/java/com/apollographql/android/impl/ApolloClient.java

```diff
@@ -147,6 +156,12 @@ public void toJson(JsonWriter writer, T value) throws IOException {
       return this;
     }
 
+    public Builder dispatcher(@Nonnull ExecutorService dispatcher) {
```

Comment:
I might be missing something here, but is there any reason for us to expose this API if the dispatcher would only be used for internal stuff like enqueuing and cache writes?